src/grt/src/fastroute/src/maze.cpp

```diff
@@ -1097,59 +1098,51 @@ void FastRouteCore::reInitTree(const int netID)
   checkAndFixEmbeddedTree(netID);
 }
 
-float getCost(const int i,
-              const float logis_cof,
-              const float cost_height,
-              const int slope,
-              const int capacity,
-              const int cost_type)
+double FastRouteCore::getCost(const int i,
```

Comment:
The parameter `i` could have a more descriptive name. Looking at the usage of this functions, the code is passing a `pos`, but it doesn't seem to be a position relative to the grid.

Reply:
It is probably a position related to the heap.